portal/templates/home.html

```diff
@@ -1,5 +1,39 @@
 {% extends "layouts/base.html" %}
 
 {% block content %}
+<div>
```

Comment:
I think we could probably remove this div, either that or add a more semantic tag